scripts/devenv.sh

```diff
@@ -3,7 +3,14 @@
 set -eu -o pipefail
 set -x
 
-docker build --pull -t acs-engine .
+docker build --pull -t \
```

Comment:
by adding these build arguments, what happens if the variables like $http_proxy do not exist (for non corp users)